annotate.el

```diff
@@ -2843,6 +2843,15 @@ sophisticated way than plain text."
   'follow-link t
   'help-echo "Click to replace annotation")
 
+(defun annotate-info-setup (file-or-node buffer)
+  "Display Info node FILE-OR-NODE in BUFFER.
+
+Compatibility wrapper for the function `info-setup' and `info-pop-to-buffer'."
+  (if (version< "30" emacs-version)
+      (info-pop-to-buffer file-or-node buffer)
+    (with-no-warnings
+      (info-setup file-or-node buffer))))
```

Comment:
I've fixed the compatibility issue here.

Reply:
That's great!